src/utils/misc.js

```diff
@@ -232,7 +232,7 @@ export default class Misc {
         channels.map(
             (c) => {
                 url+= (i !== 0 ? ',' : '') + (!c.active ? '-' : '') + (++i) +
-                 "|" + c.window.start + ":" + c.window.end + "$" + c.color;
+                 "|" + c.start + ":" + c.end + "$" + c.color;
```

Comment:
@jburel the property access was right as it was. this method is used for saving and copy both of which don't work at the moment.